docs/README.mdx

```diff
@@ -63,6 +63,6 @@ Visit the [faucet][faucet] to obtain testnet tokens for development purposes.
     findSidebarItem('/node/run-your-node/paratime-node'),
     findSidebarItem('/node/run-your-node/paratime-client-node'),
     findSidebarItem('/node/web3'),
-    findSidebarItem('/dapp/emerald/'),
-    findSidebarItem('/dapp/cipher/'),
+    findSidebarItem('/dapp/tools/other-paratimes/emerald/'),
+    findSidebarItem('/dapp/tools/other-paratimes/cipher/'),
```

Comment:
```suggestion
    findSidebarItem('/dapp/tools/other-paratimes/'),
```

Maybe Pontus X will also land here so let's just have a link to the parent folder.